docs/api/ref/schemas/country-code.yaml

```diff
@@ -0,0 +1,82 @@
+openapi: 3.1.0
+components:
+  schemas:
+    CountryCode:
+      type: string
+      enum:
+        &countryCodes [
```

Comment:
Those country codes are only the country codes available in the Eco-Score I think, so it would be good to change the file name to ecoscore-country-code.yml I think. In the API we use country codes in many other fields, and they can include other countries.

Reply:
done